### PR TITLE
Update faqs.md

### PR DESCRIPTION
**Pull request by mariapao**
I just made a few changes regarding the application process and the community participation

### CODE REVIEW

File: faqs.md [hunk 1]
```diff
@@ -58,7 +65,7 @@ We’re in a privileged position to push forward and raise awareness about great
 
 ### **Community**
 
-> Once we have the [Aragon Voting App](../dev/apps/voting.md) and a feedback mechanism from the community, all of ANT holders will get to participate in the decision making!
+We are working on transitioning Aragon’s governance from a highly centralized model to a more decentralized one and we will be experiment with community participation in the Nest program.
```
Comment (Smokyish) [inline faqs.md]: Should be either "be experimenting" or "will experiment"

File: faqs.md [hunk 2]
```diff
@@ -21,13 +21,20 @@ If there’s already an [existing proposal for grants](https://github.com/aragon
 - Make sure the [Proposals for grants](https://github.com/aragon/nest/issues) doesn’t have an existing proposals for something similar to your proposal. If there is an existing one that is close to your idea, join in the discussion of that Proposal to see if yours could be integrated into the existing one, or if you should create a new proposal
 - If nothing similar exists, create [a new Proposal for grants](https://github.com/aragon/nest/issues/new) at the [Nest Repository](https://github.com/aragon/nest/) and fill it out in detail following the [Guide for submitting a proposal for grants](guides/guide_for_submitting_a_proposal_for_grants.md)
 
-## **Who will be deciding on how the grants are distributed?**
-The funding will be described in the Proposal of the grant. The total amount, how many portions will it be divided into, timetable and milestones will be up for discussion.
+## **How does the application process works?**
+The application process has two stages:
+
+1. The proposal stage (see above). The main focus in this stage is the discussion about the idea/project and whether it should be considered for a grant. Once the proposal is approved we move to the second stage.
```
Comment (Smokyish) [inline faqs.md]: "whether or not it" could be a better wording

Comment (Smokyish) [reply faqs.md]: Also lines 9, 17 and 22 have broken links to guides, they have been moved so the links should be fixed.

Comment (Smokyish) [reply faqs.md]: And L12 has broken links to `team.md` & `roadmap.md` templates.

File: faqs.md [hunk 3]
```diff
@@ -21,13 +21,20 @@ If there’s already an [existing proposal for grants](https://github.com/aragon
...
+
+1. The proposal stage (see above). The main focus in this stage is the discussion about the idea/project and whether it should be considered for a grant. Once the proposal is approved we move to the second stage. 
+
+2. The request for funding (rff) stage (see above). The main focus in this stage is the discussion about the team, timeline/roadmap, milestones and deliverables. Here we take the final decision on whether to fund the team.
 
-Placeholder and Aragon are the ones making the **initial** decisions. But as soon as the [Aragon Voting App](../dev/apps/voting.md) and it’s accompanying signaling mechanism is ready, **the community will have the final decision** on which proposals are granted funding.
+There are no deadlines for applications. The whole application process can take from 2 weeks up to a month but the length of the process depends on a case by case basis. 
+
+## **Who will be deciding on how the grants are distributed?**
+The funding will be described in the proposal and rff stages of the grant. The total amount, how many portions will it be divided into, timetable and milestones will be up for discussion with the relevant team.
```
Comment (Smokyish) [inline faqs.md]: For clarity, let's just write out "request for funding" instead of the abbreviation.

File: faqs.md (hunk 1 above)
Comment (mariapao) [reply faqs.md]: Oh sure that's a typo

File: faqs.md (hunk 2 above)
Comment (mariapao) [reply faqs.md]: can you fix that?

File: faqs.md (hunk 3 above)
Comment (mariapao) [reply faqs.md]: Ok!

**Pull request merged**